Cargo.toml

```diff
@@ -53,7 +53,7 @@ openat = "0.1.21"
 openat-ext = "^0.2.2"
 openssl = "0.10.38"
 os-release = "0.1.0"
-ostree-ext = "0.3.0"
+ostree-ext = { git = "https://github.com/ostreedev/ostree-rs-ext", branch = "main" }
```

Comment:
The commit message says 0.5.0 but this is pointing to the main branch. Are you planning to cut another release and update this?

Reply:
Yeah I'll do that soon, I wanted to have green CI here, which I think we mostly have.  But then I started doing more API changes, so will land those first.

Reply:
And that's done!